Sources/SwifterMedia.swift

```diff
@@ -60,22 +71,56 @@ public extension Swifter {
         let parameters = ["command": "FINALIZE",
                           "media_id": mediaId]
         self.postJSON(path: path, baseURL: .upload, parameters: parameters, success: { (json, response) in
+            guard let obj = json.object, obj.keys.count > 0 else {
+                let error = SwifterError(message: "Bad Response for Multipart Media Upload",
+                                         kind: .invalidMultipartMediaResponse)
+                failure?(error)
+                return
+            }
+            if let processingInfo = json["processing_info"].object, let state = processingInfo["state"]?.string {
```

Comment:
This looks very similar to the response handler in `statusUpload(mediaId:success:failure:)`, can you wrap this into a separate private helper method so we don't have code duplication here?

Reply:
2b53ea2
Modified my code.